tests/phpunit/tests/user/retrievePassword.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Test retrieve_password, in wp-includes/user.php
+ *
+ * @group user
+ */
+class Tests_User_RetrievePassword extends WP_UnitTestCase {
+
+	public function set_up() {
+		parent::set_up();
+
+		// Create the user.
+		$this->user = self::factory()->user->create_and_get(
+			array(
+				'user_login' => 'jane',
+				'user_email' => 'r.jane@example.com',
+			)
+		);
+	}
+
+	public function retrieve_password_reset_notification_email() {
```

Comment:
```suggestion
	public function test_retrieve_password_reset_notification_email() {
```
Test method names should be prefixed with `test_` to allow PHPUnit to recognise them as tests.

Reply:
Done!